### ISSUE TITLE
detail :  "WorkGroupUser matching query does not exist."

### ISSUE THREAD

**Issue by Kattenelvis** [Gofven/flowback-backend#143]
This error occurs with API group/1/schedule/create whenever there is a workgroup being sent in the request. Even though I'm a moderator of the group. This shouldn't happen.

**Comment by Gofven** [comment]
Can i have a traceback c:?

**Comment by Kattenelvis** [comment]
I don't think it had a traceback. Although I'm unable to recreate the bug now.

**Comment by Kattenelvis** [comment]
Ok I managed to reproduce it. The bug occurs whenever one tries to create an event in a workgroup one is not a member off. But if I'm an admin, I think I should be allowed to do this anyways

**Comment by Gofven** [comment]
Check if still relevant
